Respect verbosity

mceachen opened the pull request:
This fixes #16. (At least) on my machine, adding the verbose flag (`/V`) made `tasklist` slow down dramatically. This patch only adds the /v option if the verbose opt is set, and fixes `tasklist` (again, at least on my machine).

With the PR:
```
$ time node -e 'require(".")().then(arr => console .log("found " + arr.length + " procs"))'
found 231 procs

real    0m0.717s
user    0m0.000s
sys     0m0.031s
```

Without this PR, off of master:

```
$ time node -e 'require(".")().then(arr => console.log("found  " + arr.length + " procs"))'
found 231 procs

real    0m5.558s
user    0m0.015s
sys     0m0.015s
```

mceachen committed:
Only add the /v flag if opts.verbose is set

mceachen committed:
node 4 isn't supported by latest npm anymore.

MarkTiedemann commented:
Not quite sure whether @sindresorhus wants the `package.json` URL to be changed...

Otherwise, LGTM.

mceachen commented:
> Not quite sure whether @sindresorhus wants the package.json URL to be changed...

My package.json linter complained, so I fixed it, but that change is unrelated to this bug, so I rebased that diff out of this PR just now.

The other unrelated change of removing node 4 from the build matrix was necessitated by your requirement of updating npm to the latest version, which doesn't seem to run on node 4. You'll need to either remove the npm update, or remove node 4, and given it's past EOL, I thought removing it was the safest/most palatable option.
